docs/current/sailbot_workspace/setup.md

```diff
@@ -3,7 +3,32 @@
 This workspace can be run on Windows, Linux, or macOS, but is the easiest to set up and performs the best on
 [Ubuntu](https://ubuntu.com/){target=_blank} and [its derivatives](https://distrowatch.com/search.php?basedon=Ubuntu){target=_blank}.
 
-## 1. Set up prerequisites
+Throughout this documentation, there are references to running VS Code commands, tasks, and launch configurations.
+Expand the box below to learn how to do so:
+
+??? note "Running VS Code commands, tasks, and launch configurations"
```

Comment:
Perhaps this might belong in the "How To's" section rather than a note since running VS code commands is essential to using the workspace. I would put it as the first section in that page and then link to it here.

Reply:
I put it in the main setup page because it is essential, not just an additional thing that I could offload to the how to's page. I made it collapsible because some people might know how to run these already. What do you think about this line of reasoning?